app/models/katello/concerns/host_managed_extensions.rb

```diff
@@ -271,9 +271,9 @@ def import_enabled_repositories(repos)
       def import_module_streams(module_streams)
         streams = {}
         module_streams.each do |module_stream|
-          stream = AvailableModuleStream.where(name: module_stream["name"],
+          stream = AvailableModuleStream.create_or_find_by!(name: module_stream["name"],
```

Comment:
Do you need the exclamation here ?

Reply:
We used one in `first_or_create!` so I just kept it.

Reply:
```ruby
irb(main):006:0> ::Katello::ContentCredential.create_or_find_by!(name: "baz", organization: org, content_type: "cert", content: "justin")
=> #<Katello::ContentCredential id: 1, name: "baz", organization_id: 1, content: "justin", created_at: "2023-01-12 20:05:28", updated_at: "2023-01-12 20:05:28", content_type: "cert">
irb(main):007:0> ::Katello::ContentCredential.create_or_find_by!(name: "baz", organization: org, content_type: "cert", content: "justin")
Traceback (most recent call last):
        2: from lib/tasks/console.rake:5:in `block in <top (required)>'
        1: from (irb):7
ActiveRecord::RecordInvalid (Validation failed: Name has already been taken)
```
vs
```ruby
irb(main):009:0> ::Katello::ContentCredential.create_or_find_by(name: "baz", organization: org, content_type: "cert", content: "justin")
=> #<Katello::ContentCredential id: nil, name: "baz", organization_id: 1, content: "justin", created_at: nil, updated_at: nil, content_type: "cert">
```
I prefer the exclamation version.